lib/msal-node/docs/sni.md

```diff
@@ -32,26 +32,32 @@ The snippet below demonstrates how to initialize MSAL for SNI authentication:
 
 ```js
 var msal = require('@azure/msal-node');
-
-const privateKey = `-----BEGIN PRIVATE KEY-----
-/**** PRIVATE KEY CONTENT ***/
------END PRIVATE KEY-----`; 
-
-const x5c = `-----BEGIN CERTIFICATE-----
-/ *** CERTIFICATE CONTENT1 ***/
------END CERTIFICATE-----
-
------BEGIN CERTIFICATE-----
-/ *** CERTIFICATE CONTENT2 ***/
------END CERTIFICATE-----
-`; // public key certificate chain. May contain more than 1 certificate sections
+// Due to security reasons, secrets should not be hardcoded.
+// The dotenv npm package can be used to store secrets in a .env file (located in project's root directory)
+// that should be included in .gitignore.
+require('dotenv').config(); // process.env now has the values defined in a .env file
+
+// -----BEGIN PRIVATE KEY-----
```

Comment:
We should be able to remove the cert structure here, may cause confusion as we are no longer encouraging to add the content as is. Warn about platform specific line endings that probably will impact this when saving the cert into the .env file.